README.md

```diff
@@ -216,39 +216,14 @@ To run the census functional tests within the cypress UI:
 
 ./node_modules/cypress/bin/cypress open
 
-### Deployment with elastic beanstalk
+### Deployment with [Helm](https://helm.sh/)
 
-You will need to install the EB CLI tools using PIP.
+To deploy this application with helm, you must have a kubernetes cluster already running.
 
-```
-pip install --user awsebcli        # install the eb cli tools
-```
-
-The Elastic Beanstalk CLI requires the presence of a requirements.txt file. To generate one with Pipenv use the following:
-```
-pipenv lock -r > requirements.txt
-```
-
-Initialise the project using the command
-
-```
-eb init --region eu-west-1
-```
-
-This will launch a wizard asking for the AWS credentials and some questions about the environment to create.
-
-`eu-west-1` is the name for Ireland.  I chose the default application name.
-
-Once completed, you can then deploy the application using the following command:
-
-```
-eb create
-```
-
-This will create the environment and spin up the application . Once the application has deployed you can use the following command to open it in a browser
+To deploy to a cluster that has Helm running you can run the follwoing command
 
 ```
-eb open
+helm install --name survey-runner ./helm --set projectId=PROJECT_ID --set submissionBucket=BUCKET_NAME 
```

Comment:
I think we should add the same instructions as found in census-eq-terraform to install helm and execute `helm init` before running this.